### DISCUSSION TITLE
Savvycan V213 will not enable and send out data most of the time.

### DISCUSSION THREAD

**Discussion by Pauldoof** [collin80/SavvyCAN#689] [Q&A]
I have two pieces of hardware Arduino due and a CSS Electronics CL1000 that I have connected to the SavvyCAN software V213.
The problem I have is that the devices connect OK via the software I setup the speed and press the enable bus.
When I select send frames/Play back, then load file and play the console shows data transmitting and the main SAVVYCAN window shows data transmitting but both DUE and CL1000 dont output most of the time.
The odd time it might output but mostly not.
(I have an Oscilloscope measuring for any activity)
Does anyone have any Ideas to get the enable to work?
Thanks

**Comment by MatinF** [comment]
I suggest reaching out via our contact form if this has not yet been resolved. Martin, CSS Electronics:
https://www.csselectronics.com/pages/contact-us

**Comment by MatinF** [comment]
If others encounter this: The most likely reason is if your device Configuration File (`CONFIG.INI`) has set the silent mode to true - in which case the device is deliberately unable to transmit any data onto the CAN bus. You will need to change this in the configuration file.
